src/state/GameState.ts

```diff
@@ -312,6 +312,22 @@ export class GameState {
         return this.players.filter((player) => player.teamName === teamName);
     }
 
+    public getMaxActivePlayers(): number {
```

Comment:
We should add some tests in `GetActivePlayerTests.ts` to make sure the method behaves as intended. Some tests could be
- Correct answer when no subs/joins/leaves are done
- Correct answer when some players leave
- Correct answer when some players join
- Correct answer when players leave than join (or join than leave)